app/views/tag_wranglers/show.html.erb

```diff
@@ -17,6 +17,12 @@
       <p><%= ts("Your lack of fandoms makes the wrangulator sad. Fandoms can be assigned via the") %> <%= link_to ts('tag wranglers page'), tag_wranglers_path %>.</p>
     <% else %>
       <p><%= ts('Fandoms assigned to you via the ') %> <%= link_to ts('tag wranglers page'), tag_wranglers_path %>. <%= ts('The unwrangled count is the number of unwrangled tags which were used on works in the given fandom.') %></p>
+    <% end %>
+    <% if (@wrangler == @current_user || logged_in_as_admin?) && @wrangler.last_wrangling_activity %>
```

Comment:
Could you switch this to just `logged_in_as_admin`? I mentioned to the Tag Wrangling chairs that we were thinking about putting this in the next release, but they want to give wranglers a lot of advance warning about wrangler-facing changes, so... it's either switch this to just admin-visible or wait. Just admin-visible seems like the better option.

Reply:
Sounds reasonable to me! Is there a follow up story to add this back I can link (unfortunately I don't have permissions to create one if needed) before https://otwarchive.atlassian.net/browse/AO3-6028 or should I just update that one to include this?

Reply:
I checked back in to see if they wanted exposing the date handled as part of AO3-6028 or separately, and... it turns out they have now written the email. So I'm really sorry, but could you revert the last commit and return it to being visible to the relevant wrangler?